codes/quantum/qubits/xzzx.yml

```diff
@@ -0,0 +1,31 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: xzzx
+physical: qubits
+logical: qubits
+
+name: 'XZZX surface code'
+introduced: '\cite{arXiv:2009.07851}'
+
+description: 'A family of stabilizer codes whose generators are \(XZXZ\) Pauli strings associated, clock-wise, to the vertices of each face of a two-dimensional lattice (with a qubit located at each vertex of the tessellation)'
+
+protection: 'As stabilizer \([[n=O(d^2), k=O(1), d]]\).'
+
+features:
+  threshold:
+  - '\(50\%\) with pure \(X, Y\) or \(Z\) errors.'
```

Comment:
Need to specify the decoder.  Should also mention the symmetries of the code at infinite bias.  The errors are constrained to move along lines so that the matching problem becomes a series of decoupled repetition codes.

Reply:
Added notes about error decomposition and added details here